src/views/self/view_self.cpp

```diff
@@ -62,6 +64,12 @@ namespace big
 
 		ImGui::EndGroup();
 
+		components::player_command_button<"lckick">(g_player_service->get_self());
+		if (ImGui::IsItemHovered())
+		{
+			ImGui::Text("Lose connection to others. Only works when not host.");
+		}
+
```

Comment:
Remove, this has been patched.

It's possible that this is removed automatically by the merge with master.